### DISCUSSION TITLE
Rearrange column position

### DISCUSSION THREAD

**Discussion by muksidin** [Budibase/budibase#8917] [Help] [answered]
Hi All,
How to rearrange column position in database. In this case i use Bubibase DB.
For example,i have some columns ie A, B, C. Then i edit column A, so the structure is changed to be B, C, A. How to make column A is the first.
Thanks

**Comment by melohagan** [comment, marked as answer]
No way to do this at the moment; open issue here: https://github.com/Budibase/budibase/issues/3408

I have seen a design around this, so it will come some day ;)

**Reply by muksidin** [reply]
Great, thanks.
